selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/packages/RenamePackageTest.java

```diff
@@ -132,10 +132,10 @@
           + "}\n";
 
   private static String TEST9_HIERAR_OUT =
-      "package your;\n"
+      "package my2;\n"
```

Comment:
I don't understand why we need these changes.

Reply:
I thought it's wrong behavior to rename some package to the new one which already exists. But Exlipse allows this operation. So I've also added this possibility to che and reverted this changes.